kura/org.eclipse.kura.cloudconnection.eclipseiot.mqtt.provider/src/main/java/org/eclipse/kura/internal/cloudconnection/eclipseiot/mqtt/cloud/MessageHandlerCallable.java

```diff
@@ -94,6 +94,8 @@ public Void call() throws Exception {
 
         String requestId = (String) this.kuraMessage.getMetric(METRIC_REQUEST_ID);
         if (requestId == null) {
+            logger.error("Request Id is null");            
```

Comment:
Use a lower logging level since it's not critical for execution. I think `debug` would be more apt.

For performance reasons I would check for the log level and create the string only if we're at the correct one. See here for reference.

```suggestion
        if(logger.isDebugEnabled()) {
            logger.error("Request Id is null"); 
        }
```